src/richie/apps/courses/templates/courses/cms/fragment_organization_main_logo.html

```diff
@@ -0,0 +1,12 @@
+{% load cms_tags %}
+{% if current_page.publisher_is_draft %}
+    {% if organization.check_publication is True %}
+        {% show_placeholder "logo" organization.public_extension.extended_object %}
```

Comment:
We need to display a default image when the organization has none defined.

Reply:
But what kind of image we could push when there is none ?